docs/bipoc.rst

```diff
@@ -0,0 +1,82 @@
+Our #bipoc channel
+==================
+
+This is a space dedicated to supporting BIPOC (Black, Indigenous, and People of Color) members within our community.
+
+Background
+----------
+
+As with many other professions, BIPOC remain painfully underrepresented in the tech industry, including but not limited to technical communication, software development, and open source.
+
+According to Open MIC's 2017 Report, `Breaking the Mold: Investing in Racial Diversity in Tech`_:
+
+.. _`Breaking the Mold: Investing in Racial Diversity in Tech`: http://breakingthemold.openmic.org/
+
+* Black people, Latinos and Native Americans are underrepresented in tech by 16-to-18 percentage points compared with their presence in the U.S. labor force overall. Black people and Latinos each comprise just 5.3 percent of the Professionals category in U.S. tech industry labor data.
+* While Asians are represented at a higher rate in the tech workforce than the private sector overall, white people are 1½ times more likely than Asians to rise to an executive rank.
+* Among people of color who do enter the industry, many report isolation, discrimination and toxic work environments that prompt them to take their talent else-where. People of color leave tech at more than 3.5 times the rate of white men.
+
+To shrink racial disparities and to build equity, we offer this space as BIPOC tech writers to:
```

Comment:
Made a comment in the Slack channel as well, but wanted to ask about `tech writers` vs., say, `documentarians`. The latter would be more consistent with how we refer to ourselves across WtD. _But_, to Cheryl's point from yesterday, those who aren't already in the WtD community may not be familiar with the term and want to better understand who we are and what we all do. 

Hm, I suppose we could also link out to the [Documentarians](https://www.writethedocs.org/documentarians/) page for their reference?

Reply:
```suggestion
To shrink racial disparities and to build equity, we offer this space as BIPOC :doc:`/documentarians`. (which includes technical writers) to:
```